src/opnsense/scripts/routes/ipv6_gateways.php

```diff
@@ -0,0 +1,21 @@
+#!/usr/local/bin/php
```

Comment:
we would rather fold this special case return into an existing script. duplicating functionality for a single IP family is not ideal and we can easily build a plugin-based wrapper that can be called with `pluginctl`.

Reply:
Please point me in the right direction, where you want it, I'm not familiar with this.
I was trying to fetch the info on gateways directly from system, but gateway names are not there. So they are extracted from possibly non-applied config instead.